pages/api/user/profile.ts

```diff
@@ -11,15 +11,11 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
   }
 
   // Get user
-  const user = await prisma.user.findUnique({
-    where: {
-      email: session.user.email,
+  const user = await whereAndSelect(prisma.user.findUnique, {
```

Comment:
Codacy has a fix for the issue: [Replace `prisma.user.findUnique,` with `⏎····prisma.user.findUnique,⏎···`](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7734611)
```suggestion
  const user = await whereAndSelect(
```